AmericanMedicalAssociation/AMA-style-guide
# Ewl 3283 3284 search results

## JIRA Ticket(s)
- [EWL-3283: Style Guide | Search result columns](https://issues.ama-assn.org/browse/EWL-3283)
- [EWL-3284: Style Guide | Search result list external](https://issues.ama-assn.org/browse/EWL-3284)


## Description:
Builds off of [PR 98](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/98) and [PR 101](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/101). Both Search Results - Column and Search Results - List now use the same twig template for markup so they can be implemented with a simple class change on the same markup. Creates a new layout for the 2 columns as discussed with Ashley and Chelsie in Slack. The CSS is ugly, but I am resisting the temptation to throw it out and start over.


## To Test:
- go to Organisms > Global > Search Results - Column
- verify the template matches the comp
- go to Organisms > Global > Search Results - List
- verify the template matches the comp
- go to Molecules > Layouts > Two Up Wrap Layout
- verify this creates two columns of sibling elements without wrappers in the markup


## Automated Test:
N/A


## Relevant Screenshots/GIFs:
N/A


## Remaining Tasks:
The two column layout probably needs a mobile, but I don't have a comp for this.


## Additional Notes:
N/A


---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
